app/res/drawable/login_edit_text.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <padding>8dp</padding>
+    <size android:height="48dp"></size>
```

Comment:
These are standard constants for us now, right? They should be either in a style or dimensions file, shoudln't they?